src/main/java/io/github/moulberry/notenoughupdates/util/ApiUtil.java

```diff
@@ -184,6 +184,9 @@ public CompletableFuture<String> requestString() {
 							OutputStream os = conn.getOutputStream();
 							try {
 								os.write(this.postData.getBytes("utf-8"));
+							} catch (Throwable t) {
+								t.printStackTrace();
+								throw new Error(t);
```

Comment:
Please use
```suggestion
								throw new CompletionException(t);
```
instead to avoid double boxing.

Reply:
This is now Irrelevant.